python/langsmith/client.py

```diff
@@ -489,6 +501,17 @@ def __init__(
         # Create a session and register a finalizer to close it
         session_ = session if session else requests.Session()
         self.session = session_
+        self.compress_traces = ls_utils.get_env_var("USE_RUN_COMPRESSION")
+        if self.compress_traces:
+            self.boundary = BOUNDARY
+            self.compressed_runs_buffer: io.BytesIO = io.BytesIO()
+            self.compressor_writer: zstd.ZstdCompressionWriter = zstd.ZstdCompressor(level=3).stream_writer(
```

Comment:
I'd probably use the lowest level, we want to minimize cpu usage

Reply:
With `level=1` we're only getting a compression ratio of 6.7 vs 18+, `level=2` puts us around 12. I'm open to experimenting with this to see if we still OOM with `level=1`